static/scss/partials/main.scss

```diff
@@ -2376,22 +2376,31 @@ input.input-has-error {
   width: 100%;
   padding: $spacing-lg $spacing-md 0 $spacing-md;
   margin: 0 auto;
-}
 
-.messages > div {
-  background-color: var(--grey50);
-}
+  span {
+    color: $color-white;
+    text-align: center;
+    width: 100%;
+    font-weight: 600;
+  }
 
-.messages .success {
-  background-color: var(--successGreen);
-  border-radius: $border-radius-md;
-  box-shadow: var(--dropShadowLight);
-}
+  path {
+    fill: $color-white;
+  }
+
+  .message-wrapper {
+    border-radius: $border-radius-md;
+    box-shadow: var(--dropShadowLight);
+  }
+
+  .success {
+    background-color: var(--successGreen);
+  }
+
+  .error {
+    background-color: var(--warningRed);
+  }
```

Comment:
I moved the properties from the redundant class `.home-messages` into `.messages`, formatted into SASS. I also encapsulated the `border-radius` and `box-shadow` properties into a wrapper class for clarity to avoid code repetition.

Reply:
Great.